test/orogen/test_logger.rb

```diff
@@ -3,52 +3,83 @@
 using_task_library "logger"
 
 module OroGen
-    # OroGen.logger already exists, we can't access the model for logger::Logger
-    # using the method syntax
-    logger_m = OroGen.syskit_model_by_orogen_name("logger::Logger")
-
-    describe logger_m do
+    # NOTE: RockLogger is set to the logger model in orogen/logger.rb as OroGen.logger
+    # is taken (can't access through OroGen.logger.LoggerTask)
+    describe Syskit::RockLogger do
         def deploy_subject_syskit_model
             use_deployment("rock_logger").first
         end
 
```

Comment:
@doudou, I had to keep this part, otherwise the tests would break with `NoMethodError: undefined method 'project_name' for nil:NilClass`